flake/default.nix

```diff
@@ -0,0 +1,16 @@
+{
+  imports = [
+    ./checks.nix
+    ./ci.nix
+    ./devshells.nix
+    ./overlays.nix
+    ./packages
+  ];
+
+  perSystem = {pkgs, ...}: {
+    formatter = pkgs.alejandra;
+    _module.args = {
+      lua = pkgs.lua5_1;
```

Comment:
Idk if this is gonna end up as an arg to neovim-unwrapped? Also don't know if neovim cares about being compiled with lua vs luajit. But check :version just to be sure

Reply:
Yeah, except the packages in flake/packages are also perSystem. More of a callout. In fact I will check it real quick!